README.md

```diff
@@ -1,34 +1,28 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Clerk - Next.js starter repository
 
-## Getting Started
+This repository shows an example use case for how you can authenticate your users using [Clerk](https://www.clerk.dev/?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter) as an authentication provider on your Next.js application.
 
-First, run the development server:
+## Running the starter
 
-```bash
-npm run dev
-# or
-yarn dev
-```
+You have two option to run the starter, we recommend the one click deployment with Vercel. This way, environment variables are set up for you automatically and it's a much easier experience.
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+### Option 1 - One click deployment with Vercel
 
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+You can use the following deployment button from Vercel, [this article on our blog](https://clerk.dev/blog/clerk-vercel-one-click-deployment?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter) guides you step to step on how to do so.
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/clerkinc/clerk-nextjs-starter&integration-ids=oac_7uYNbc9CdDAZmNqbt3LEkO3a&external-id=starter)
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+### Option 2 - Run manually
 
-## Learn More
+To run the example locally you need to:
 
-To learn more about Next.js, take a look at the following resources:
+1. Click [**"Use this template"**](https://github.com/clerkinc/clerk-nextjs-starter/generate) to create a copy of this repository on your GitHub account.
+2. Clone your repository locally.
+3. Sign up for a Clerk account at [Clerk.dev](https://www.clerk.dev/?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter).
+4. Setup the required Clerk Frontend API variable from your Clerk project as shown at [the example env file](./.env.local.sample).
```

Comment:
We should add a step to describe that we need to create an instance.

Reply:
Added a step!